draft-linker-diem-requirements.md

```diff
@@ -55,6 +55,45 @@ TODO Introduction
 
 ## Requirements
 
+### Authenticity
+
+Validators MUST be able to verify that an emblem was issued for the respective bearer, issued by the claimed issuer (where applicable), and that none of its associated data was changed.
+
+Use cases that have this requirement:
+
+### Issuance and Authorization
+
+Validators MUST be able to verify that an emblem issuer was authorized to issue the emblem.
+Authorizing parties MAY limit what emblems an issuer can issue, e.g., they MAY limit issuance to certain bearers.
+
+Use cases that have this requirement:
+
+#### Decentralization
+
+Anyone MUST be able to authorize an issuer.
```

Comment:
I'm not sure what this means.  Could you expound and/or provide an example?

Reply:
Thanks. I think I understand you to mean that validators/consumers can authorize who they want (i.e., their equivalent of a set of root CAs)?  Does this also mean that authorized issuers can also extend that authorization to whom they want?  I'm not opposed to these, but if this is what you mean, then let's spell it out that way with something like the following:

> There is no global registry of authorized issuers.  Issuers are authorized independently by validators and other issuers.

Reply:
I don't think that's entirely what I mean, but it seems related.

Basically, I want to require that there is no one who tells someone who an auhtorizing party is (by the spec). For example, a draft could say: "There is a IANA registry for DIEM-authorizing parties and a DIEM designated experts group decides who is in that registry and validators must only accept emblems authorized by one of these issuers." This would be a straightforward violation of that requirement.

If I decide tomorrow that I want to authorize someone to display digital emblems, I should be able to do that with reaonsable effort. I should not be required to ask for permission from a fixed authority, or I should not be required to be in some registry or similar.